src/components/TextInputBase/TextInputBase.style.ts

```diff
@@ -3,40 +3,59 @@ import { Theme } from 'theme';
 import { rem } from 'theme/utils';
 import { DEFAULT_SIZE, getTextFieldSize } from 'utils/size-utils';
 
+import { getDisabled, getHover, getPressed } from '../../theme/states';
+import { textInputConfig } from './config';
 import { Props } from './TextInputBase';
 import { LABEL_TRANSFORM_LEFT_SPACING } from 'components/Label/Label.style';
 
+export const flexContainer = () => (): SerializedStyles =>
+  css({
+    display: 'flex',
+    alignItems: 'center',
+    position: 'relative',
+    width: '-webkit-fill-available',
```

Comment:
Not a fan of using a non standard value for width here. Can we make this work in a standards compliant way? See here for standardized values: https://developer.mozilla.org/en-US/docs/Web/CSS/width#browser_compatibility